### PR TITLE
Remove v1beta3

### PR DESCRIPTION
**Pull request by thockin**
@bgrant0607 @nikhiljindal

totally untested - Nikhil, feel free to close this and do your own thing.  I just want to spur it on.

### CODE REVIEW

**Comment by bgrant0607** [comment]
Related cleanup: #10892

File: pkg/api/testing/fuzzer.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 	docker "github.com/fsouza/go-dockerclient"
-	"github.com/google/gofuzz"
```
Comment (nikhiljindal) [inline pkg/api/testing/fuzzer.go]: Is this intended?
Shippable shows failures like the following:
```pkg/api/testing/fuzzer.go:38: undefined: fuzz```

**Comment by nikhiljindal** [comment]
Looks good.
Just get the tests to pass.

File: plugin/pkg/admission/resourcequota/admission.go [hunk 2]
```diff
@@ -176,9 +175,7 @@ func IncrementUsage(a admission.Attributes, status *api.ResourceQuotaStatus, cli
 	obj := a.GetObject()
 	// handle max counts for each kind of resource (pods, services, replicationControllers, etc.)
 	if a.GetOperation() == admission.Create {
-		// TODO v1beta1 had camel case, v1beta3 went to all lower, we can remove this line when we deprecate v1beta1
-		resourceNormalized := strings.ToLower(a.GetResource())
```
Comment (thockin) [inline plugin/pkg/admission/resourcequota/admission.go]: Removing this ToLower() caused a couple tests to fail.  This is a tightening of the implementation - are we OK with it?

Comment (bgrant0607) [inline plugin/pkg/admission/resourcequota/admission.go]: There's not a big complexity cost to keeping it, so I'd just leave it in as a user convenience.

Comment (nikhiljindal) [inline plugin/pkg/admission/resourcequota/admission.go]: Yes I think this is fine. We do not allow camelCase at many other places (looks like we deleted them when we stopped supporting beta1,2).

@bgrant0607 to confirm.

Comment (thockin) [inline plugin/pkg/admission/resourcequota/admission.go]: @bgrant0607 note that it is very inconsistent - as far as I know the URL paths are not treated the same way

Comment (nikhiljindal) [inline plugin/pkg/admission/resourcequota/admission.go]: What if users start depending on it. Will they expect us to support camelCase at all other places as well?

Comment (bgrant0607) [inline plugin/pkg/admission/resourcequota/admission.go]: Ok, remove it. Consistency is key.

**Comment by thockin** [comment]
This branch is just waiting for shippable now.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by thockin** [comment]
Shippable is a data race 1/2 passed/

**Comment by nikhiljindal** [comment]
Needs rebase

**Comment by thockin** [comment]
I vote this PR as "most likely to get rolled back" and also "most likely to need a rebase".

@vishh

**Comment by mikedanese** [comment]
Fire in the hole!

**Pull request merged**